cluster/gce/config-default.sh

```diff
@@ -86,7 +86,7 @@ fi
 # you are updating the os image versions, update this variable.
 # Also please update corresponding image for node e2e at:
 # https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/image-config.yaml
```

Comment:
is this comment still valid? Link is broken and the image config in test-infra don't need to be updated with this variable update. Wdyt?

Reply:
I don't  think this is still valid.

Reply:
I believe image-config.yaml was removed in 1.19 with https://github.com/kubernetes/kubernetes/pull/91471